OrbitGl/PickingManager.cpp

```diff
@@ -8,52 +8,77 @@
 
 //-----------------------------------------------------------------------------
 PickingID PickingManager::CreatePickableId(Pickable* a_Pickable) {
+  absl::MutexLock lock(&mutex_);
   ++m_IdCounter;
-  PickingID id = PickingID::Get(PickingID::PICKABLE, ++m_IdCounter);
```

Comment:
Oh, was this incremented twice here by accident?

Reply:
Yes :-)